### ISSUE TITLE
load dump don't works

### ISSUE THREAD

**Issue by qrp73** [devydd/UT210E-EEPROM-Editor#1]
I'm trying to open bin file with eeprom dump, but nothing happens. There is error in console:
```
UT210E_EEPROM_editor.html:1405

Uncaught TypeError: hexDump.replaceAll is not a function
    at getHexDumpText (UT210E_EEPROM_editor.html:1405)
    at processEEPROM (UT210E_EEPROM_editor.html:1664)
    at FileReader.reader.onload (UT210E_EEPROM_editor.html:1302)
getHexDumpText @ UT210E_EEPROM_editor.html:1405
processEEPROM @ UT210E_EEPROM_editor.html:1664
reader.onload @ UT210E_EEPROM_editor.html:1302
load (async)
readEEPROMFile @ UT210E_EEPROM_editor.html:1298
(anonymous) @ UT210E_EEPROM_editor.html:1719
```

**Comment by qrp73** [comment]
I tried to replace this line https://github.com/devydd/UT210E-EEPROM-Editor/blob/9cb3b763d8e98484709f90f6ff25a5400a90e200/src/UT210E_EEPROM_editor.html#L1405

with this one:
```
hexDump = hexDump.replace(/([\s-]+)<\/span>/gm, `</span>$1`); 
```
and at a glance it works

**Comment by devydd** [comment]
@qrp73 Thanks for the report! I cannot reproduce the issue with the latest `Chrome 90.0.4430.212 (Official Build)`. `replaceAll` [should be supported](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll) on all modern browsers... What browser are you using?

**Comment by qrp73** [comment]
replaceAll is a new feature, it is not supported on old browsers. 

.replaceAll will be available starting on Chrome 85 or Firefox 77. IE doesn't support it at all. 

If you want to use replaceAll, there is needs for browser version check and select appropriate implementation. But using replace instead of replaceAll just fix it with no need to implement a bunch of version check and other stuff. replace works on any browser.

As I said, I just replaced "replaceAll" with "replace" and it works ok on any browser, no needs to change something else.

**Comment by devydd** [comment]
That's right. Those functions have the same functionality when using global regular expression as a pattern. Fixed in e006a4a23574b43d88a2d4e9d5612280c26cf68e and published (github pages). Thanks!
